tasks/cloud_vpn/providers/rhel/initiator/configure_vpn.yaml

```diff
@@ -16,14 +16,14 @@
 
 - name: Push Libreswan configuration file to initiator
   template:
-    src: cloud_vpn/providers/rhel/initiator/{{ cloud_vpn_responder_provider }}/configure_static.j2
+    src: cloud_vpn/providers/rhel/initiator/{{ cloud_vpn_responder_provider }}/configure_static_{{ cloud_vpn_mode }}.j2
```

Comment:
Do we need to add default value for cloud_vpn_mode as main mode with a warning for using not so secure aggressive mode ? I could not find this var in document also ..or this is part of another PR ?